languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/remote-rule-filters.xml

```diff
@@ -665,7 +665,7 @@ To ignore a remote rule match, set the <marker> so that it exactly covers the te
                 <pattern>
                     <token postag="K"/>
                     <marker>
-                        <token spacebefore="no" regexp="yes">-|-|—</token>
+                        <token spacebefore="no">-</token>
```

Comment:
To fix a maven warning (and also because now that I think about it, it doesn't make sense to have something else besides the regular "-")